modules/installation-proxy-remove.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * installing/insall_config/enable-cluster-wide-proxy.adoc
+
+[id="installation-proxy-remove_{context}"]
+= Removing proxy from the cluster
+
+The `cluster` proxy object cannot be deleted. To remove proxy from a cluster,
+remove all `spec` fields from the proxy object.
+
+.Prerequisites
+
+* Cluster administrator permissions
+* {product-title} `oc` CLI tool installed
+
+.Procedure
+
+. Use the `oc edit` command to modify the proxy:
++
+----
+$ oc edit proxy/cluster
+----
+
+. Remove all `spec` fields from the proxy object. For example:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Proxy
+metadata:
```

Comment:
In all my examples, I've removed the metadata specific to the running cluster, usually leaving just this:

```
metadata:
  name: cluster
```

Reply:
@bergerhoffer as @jboxman mentions, drop all the metadata fields except `name: cluster`.